frontend/frontend-react/src/components/template/Header.css

```diff
@@ -0,0 +1,29 @@
+header.header {
+    position: -webkit-sticky;
+    position: sticky;
```

Comment:
The `position` property is being overwritten. You should use just one of them in situations like that.